### PR TITLE
doc(php): add procedure to upgrade to php 7.4 on 20.10.x

### PR DESCRIPTION
**Pull request by kduret**
## Description

add procedure to upgrade php on 20.04.x

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: fr/administration/secure-platform.md [hunk 1]
```diff
@@ -574,6 +574,38 @@ vim /opt/rh/httpd24/root/etc/httpd/conf.d/10-centreon.conf
...
+sh /usr/share/centreon/bin/migrate_php73.sh
+```
+
+La sortie devrait correspondre à ceci:
+
+```
+[INFO] Installing dependencies for PHP 7.4
+...
+[INFO] Copying php-fpm configuration from 7.3 to 7.4
+[INFO] Copying php configuration from 7.3 to 7.4
```
Comment (kduret) [inline fr/administration/secure-platform.md:604]:
```suggestion
[INFO] Copying php-fpm configuration from 7.2 to 7.4
[INFO] Copying php configuration from 7.2 to 7.4
```

File: en/administration/secure-platform.md [hunk 2]
```diff
@@ -707,6 +707,38 @@ vim /opt/rh/httpd24/root/etc/httpd/conf.d/10-centreon.conf
...
+sh /usr/share/centreon/bin/migrate_php73.sh
+```
+
+The output should look like this:
+
+```
+[INFO] Installing dependencies for PHP 7.4
+...
+[INFO] Copying php-fpm configuration from 7.3 to 7.4
+[INFO] Copying php configuration from 7.3 to 7.4
```
Comment (kduret) [inline en/administration/secure-platform.md:737]:
```suggestion
[INFO] Copying php-fpm configuration from 72 to 7.4
[INFO] Copying php configuration from 7.2 to 7.4
```

File: en/administration/secure-platform.md [hunk 3]
```diff
@@ -707,6 +707,38 @@ vim /opt/rh/httpd24/root/etc/httpd/conf.d/10-centreon.conf
...
+    * centreon-open-tickets >= 20.10.3
+> You can get the list of packages versions by using following command:
+> ```shell
+> rpm -qa | grep centreon
+> ```
+
+2. Upgrade your PHP version by using following command:
+
+```shell
+sh /usr/share/centreon/bin/migrate_php73.sh
```
Comment (kduret) [inline en/administration/secure-platform.md:728]:
```suggestion
sh /usr/share/centreon/bin/migrate_php74.sh
```

File: fr/administration/secure-platform.md [hunk 4]
```diff
@@ -574,6 +574,38 @@ vim /opt/rh/httpd24/root/etc/httpd/conf.d/10-centreon.conf
...
+    * centreon-open-tickets >= 20.10.3
+> Vous pouvez obtenir la liste des versions des paquets en utilisant la commande suivante:
+> ```shell
+> rpm -qa | grep centreon
+> ```
+
+2. Mettre à jour votre version PHP en utilisant la commande suivante:
+
+```shell
+sh /usr/share/centreon/bin/migrate_php73.sh
```
Comment (kduret) [inline fr/administration/secure-platform.md:595]:
```suggestion
sh /usr/share/centreon/bin/migrate_php74.sh
```

File: en/administration/secure-platform.md [hunk 5]
```diff
@@ -707,6 +707,38 @@ vim /opt/rh/httpd24/root/etc/httpd/conf.d/10-centreon.conf
...
+```shell
+sh /usr/share/centreon/bin/migrate_php74.sh
+```
+
+The output should look like this:
+
+```
+[INFO] Installing dependencies for PHP 7.4
+...
+[INFO] Copying php-fpm configuration from 72 to 7.4
```
Comment (kduret) [inline en/administration/secure-platform.md:736]:
```suggestion
[INFO] Copying php-fpm configuration from 7.2 to 7.4
```

**Review by jeremyjaouen — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Comment by kduret** [comment]
let's reopen it if needed

**Pull request closed without merging**